tests/test_relay.py

```diff
@@ -16,8 +16,8 @@
 import humanize
 import pytest
 
-# This file is in a subdirectory, so add the CUT's location to the path
-sys.path.append(str(Path(__file__).resolve().parent.parent.parent))
+# # This file is in a subdirectory, so add the CUT's location to the path
+# sys.path.append(str(Path(__file__).resolve().parent.parent.parent))
```

Comment:
I assume you'll remove those? And having commented them, flake8 believes you should have removed the import of `sys`...

Reply:
Um, yes.  (This PR should have been moved to draft mode when the tests didn't "just run" -- it's not quite ready for prime time, yet, although I am growing in understanding of packaging issues (and starting to develop _some_ sympathy (although, it **is** limited) for our friends on `pquisby`).)

The above was a hack driven by the original file layout.  With the new file layout, it is no longer necessary (the commenting-out was a temporary experiment, which mostly succeeded) and will be removed before this is merged.

Reply:
All better now...although, sadly, this change [reduced](https://github.com/distributed-system-analysis/file-relay/pull/8#issuecomment-1699504354) the unit test coverage....  🤣